### PR TITLE
[monitoring-kubernetes] ebpf-exporter distroless

### PR DESCRIPTION
**Pull request by aleksey-su**
## Description
ebpf_exporter are now based on minimal image and bump version to v2.3.0.

If an error occurs when starting ebpf_exporter, e.g. there are not enough modules in the kernel to start, the EbpfExporterKernelNotSupported alert will be lit. 
Possible actions to resolve the problem:
 * Built kernel with BTF type information [info](https://github.com/libbpf/libbpf?tab=readme-ov-file#bpf-co-re-compile-once--run-everywhere).
 * [Disable](https://deckhouse.io/documentation/v1/modules/340-monitoring-kubernetes/configuration.html#parameters-ebpfexporterenaled) ebpf_exporter

## What is the expected result?
We use scratch along with a minimal number of files from Alpine packages. We pass files in packages through a rsync whitelist.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: monitoring-kubernetes
type: feature
summary: ebpf_exporter image is based on a distroless image. Bump version to v2.3.0.
impact_level: default
```

### CODE REVIEW

**Comment by aleksey-su** [comment]
BTF not supported
image
image

**Comment by aleksey-su** [comment]
Normal
image
image

**Comment by aleksey-su** [comment]
image
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

File: modules/340-monitoring-kubernetes/images/ebpf-exporter/werf.inc.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+shell:
+  install:
+  - cd /entrypoint
+  - export GOPROXY={{ $.GOPROXY }} CGO_ENABLED=0 GOOS=linux GOARCH=amd64
+  - go build -ldflags="-s -w" -o entrypoint main.go
+  - chown -R 64535:64535 /entrypoint/
+  - chmod 0700 /entrypoint/entrypoint
+---
+artifact: {{ .ModuleName }}/build-artifact
+from: {{ .Images.BASE_GOLANG_21_BULLSEYE }}
```
Comment (RomanenkoDenys) [inline modules/340-monitoring-kubernetes/images/ebpf-exporter/werf.inc.yaml:22]: Please use DEV image and remove apt

File: tools/dev_images/additional_tools/alt/binary_replace.sh [hunk 2]
```diff
@@ -80,12 +80,12 @@ function relocate_item() {
   local new_place="${RDIR}$(dirname ${file})"
 
   mkdir -p ${new_place}
-  cp -a ${file} ${new_place}
+  cp -a --remove-destination ${file} ${new_place}
```
Comment (RomanenkoDenys) [inline tools/dev_images/additional_tools/alt/binary_replace.sh:83]: We need to rebuild dev_alt image here ad this is a big problem - all containers which builded on the DEV_ALT base should be restarted in the cluster

File: tools/dev_images/additional_tools/alt/binary_replace.sh [hunk 3]
```diff
@@ -80,12 +80,12 @@ function relocate_item() {
...
 
   # if symlink, copy original file too
   local orig_file="$(readlink -f ${file})"
   if [[ "${file}" != "${orig_file}" ]]; then
-    cp -a ${orig_file} ${new_place}
+    cp -a --remove-destination ${orig_file} ${new_place}
```
Comment (RomanenkoDenys) [inline tools/dev_images/additional_tools/alt/binary_replace.sh:88]: The same as above

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]

**Comment by aleksey-su** [comment]
image

**Review by RomanenkoDenys — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

**Review by name212 — APPROVED** [review]

**Review by nabokihms — APPROVED** [review]

**Pull request merged**